packages/generic-application/test/fixtures/express/no-build/platformatic.app.json

```diff
@@ -0,0 +1,3 @@
+{
+  "$schema": "https://schemas.platformatic.dev/@platformatic/generic-application/1.52.0.json"
```

Comment:
I think we must have the name as a one word. Either `@platformatic/basic` or `@platformatic/generic`.

Reply:
I'll go for `basic` then.